docs/developer-guide/syntax-reference.rst

```diff
@@ -2093,70 +2093,74 @@ convention is followed.
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
-| TOPKDISTINCT           | ``TOPKDISTINCT(col1, k)`` | Stream     | Return the distinct Top *K* values for the given column and window  |
-|                        |                           |            | Note: rows where ``col1`` is null will be ignored.                  |
-+------------------------+---------------------------+------------+---------------------------------------------------------------------+
++------------------------+----------------------------+------------+---------------------------------------------------------------------+
+| Function               | Example                    | Input Type | Description                                                         |
++========================+============================+============+=====================================================================+
+| COLLECT_LIST           | ``COLLECT_LIST(col1)``     | Stream,    | Return an array containing all the values of ``col1`` from each     |
+|                        |                            | Table      | input row (for the specified grouping and time window, if any).     |
+|                        |                            |            | Currently only works for simple types (not Map, Array, or Struct).  |
+|                        |                            |            | This version limits the size of the result Array to a maximum of    |
+|                        |                            |            | 1000 entries and any values beyond this limit are silently ignored. |
+|                        |                            |            | When using with a window type of ``session``, it can sometimes      |
+|                        |                            |            | happen that two session windows get merged together into one when a |
+|                        |                            |            | late-arriving record with a timestamp between the two windows is    |
+|                        |                            |            | processed. In this case the 1000 record limit is calculated by      |
+|                        |                            |            | first considering all the records from the first window, then the   |
+|                        |                            |            | late-arriving record, then the records from the second window in    |
+|                        |                            |            | the order they were originally processed.                           |
++------------------------+----------------------------+------------+---------------------------------------------------------------------+
+| COLLECT_SET            | ``COLLECT_SET(col1)``      | Stream     | Return an array containing the distinct values of ``col1`` from     |
```

Comment:
why have all these lines changed?

Reply:
I needed to push everything to the right to fit the words in on the new entry

Reply:
This is moot as I've reverted it as old docs are deprecated